### PR TITLE
Adding component-specific self-signed cert info for IDE and Pipelines.

### PR DESCRIPTION
**Pull request by grainnejenningsRH**
Adds component-specific self-signed certificate info for workbenches and pipelines.

- Adds a _Using self-signed certificates with {productname-short} components_ section to the _Working with certificates_ assembly.
- Adds two new modules to cover component-specific info for using self-signed certificates.

Note: The KCS article referenced in both new modules is still in progress, and will be completed in the next couple of days pre-release.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: modules/using-certificates-with-data-science-pipelines.adoc [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+[id='using-certificates-with-data-science-pipelines_{context}']
+= Using certificates with data science pipelines 
+
+== Providing a CA bundle only for data science pipelines
+
+Perform the following steps to provide a Certificate Authority (CA) bundle just for data science pipelines.
+
+.Procedure
+. Create a ConfigMap with the required bundle in the same data science project or namespace as the target data science pipeline:
```
Comment (aduquett) [inline modules/using-certificates-with-data-science-pipelines.adoc:11]: Are they doing this is OpenShift > Workloads > ConfigMaps or somewhere else?

File: modules/using-certificates-with-data-science-pipelines.adoc [hunk 2]
```diff
@@ -0,0 +1,85 @@
+:_module-type: PROCEDURE
+
+[id='using-certificates-with-data-science-pipelines_{context}']
+= Using certificates with data science pipelines 
+
+== Providing a CA bundle only for data science pipelines
+
+Perform the following steps to provide a Certificate Authority (CA) bundle just for data science pipelines.
+
```
Comment (aduquett) [inline modules/using-certificates-with-data-science-pipelines.adoc:9]: Any prerequisites?

File: modules/using-certificates-with-data-science-pipelines.adoc [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
++
+----
+pod=$(oc get pod -n ${dsProject} -l app=ds-pipeline-${dspa} --no-headers | awk '{print $1}')
+----
+. Display the contents of the `/dsp-custom-certs/dsp-ca.crt` file:
++
+----
+oc -n ${dsProject} exec $pod -- cat /dsp-custom-certs/dsp-ca.crt
+----
+. Verify that your your CA bundle is present within this bundle.
```
Comment (aduquett) [inline modules/using-certificates-with-data-science-pipelines.adoc:81]: extra "your"

File: modules/using-certificates-with-workbenches.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+ifdef::cloud-service[]
+Self-signed certificates apply to workbenches that you create after configuring self-signed certificates centrally as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/working-with-certificates_certs[Working with certificates]. There is no change to workbenches that you created before configuring self-signed certificates.
+endif::[]
+ifdef::self-managed[]
+Self-signed certificates apply to workbenches that you create after configuring self-signed certificates centrally as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/working-with-certificates_certs[Working with certificates] (for disconnected environments, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/working-with-certificates_certs[Working with certificates]. There is no change to workbenches that you created before configuring self-signed certificates.
+endif::[]
+endif::[]
+====
+
+== Creating data science pipelines with Elyra and self-signed certificates
```
Comment (aduquett) [inline modules/using-certificates-with-workbenches.adoc:21]: Is this in both modules intentionally?

File: modules/using-certificates-with-data-science-pipelines.adoc (hunk 2 above)
Comment (grainnejenningsRH) [reply modules/using-certificates-with-data-science-pipelines.adoc:9]: I was told no, just the CA bundle they want to add.

File: modules/using-certificates-with-workbenches.adoc (hunk 4 above)
Comment (grainnejenningsRH) [reply modules/using-certificates-with-workbenches.adoc:21]: It is - I don't love the repetition, but it's relevant for both IDE and pipelines. Could remove from Workbenches if we think it'll be an issue?

File: modules/using-certificates-with-data-science-pipelines.adoc (hunk 3 above)
Comment (grainnejenningsRH) [reply modules/using-certificates-with-data-science-pipelines.adoc]: Good spot, updated.

File: modules/using-certificates-with-data-science-pipelines.adoc (hunk 1 above)
Comment (grainnejenningsRH) [reply modules/using-certificates-with-data-science-pipelines.adoc]: They are - have updated to add a step to log in to OCP, and indicate where to create the configmap.

**Comment by aduquett** [comment]
LGTM

**Pull request merged**